Conflict, when using this plugin with ngxs-reset-plugin (StateClear action)

Hello.
I use the `ngxs-reset-plugin` to Clear state on user logout, and reset all states to their defaults..

My code looks like this:
```
this.store.dispatch(new StateClear()); // to remove auth state information, stored in localStore (NGXS Storage plugin)
this.store.dispatch(new StateResetAll()); // to reset all states to their defaults
```

Just after executing this line:
`this.store.dispatch(new StateClear());`

I receive an error:
```
auth.state.ts:63 ERROR TypeError: Cannot read properties of undefined (reading '@@CLEAR_STATE')
    at Object.next (ngxs-labs-actions-executing.js:29:35)
    at source.subscribe.isUnsub (tap.js:17:81)
    at OperatorSubscriber._next (OperatorSubscriber.js:13:21)
    at OperatorSubscriber.next (Subscriber.js:31:18)
    at Object.next (ngxs-store.js:1521:33)
    at ConsumerObserver.next (Subscriber.js:91:33)
    at SafeSubscriber._next (Subscriber.js:60:26)
    at SafeSubscriber.next (Subscriber.js:31:18)
    at Subject.js:34:30
    at errorContext (errorContext.js:19:9)
```


The 
`@@CLEAR_STATE` is just the action type name of the `StateClear` action:
```
/**
 * Action to clear all state except given state(s)
 */
export declare class StateClear {
    static readonly type = "@@CLEAR_STATE";
    readonly statesToKeep: MetaDataModel[];
    constructor(...statesToKeep: StateClass[]);
    constructor();
}
```

But the error points here:
```
 ngxsOnInit({ patchState, getState }) {
        this.actionsExecutedSub = this.actions$
            .pipe(tap((actionContext) => {
            const actionType = getActionTypeFromInstance(actionContext.action);
            if (!actionType) {
                return;
            }
            let count = getState()[actionType] || 0; //////// <<<<<----------------- ERROR ON THIS LINE. Since the state does not exist, because it is cleared, so there is no actionType in the undefined (the actionType is '@@CLEAR_STATE'), so error come from here
// you should add a check, if there is a state.
            if (actionContext.status !== "DISPATCHED" /* Dispatched */) {
                count++;
            }
            patchState({
                [actionType]: count
            });
        }))
            .subscribe();
    }
```

Alright, I think that I am on the right way:

I have to exclude this plugin states from being cleared/reseted

```
        this.store.dispatch(
          new StateClear(ActionsExecutedState, ActionsExecutingState) // exclude these states, but they are not exported
        );
        this.store.dispatch(
          new StateResetAll(ActionsExecutedState, ActionsExecutingState) // exclude these states, but they are not exported
        );
```

But I receive this error:
```
./src/app/store/auth/auth.state.ts:9:0-97 - Error: Module not found: Error: Can't resolve '@ngxs-labs/actions-executing/lib/actions-executing.state' in '/Users/user/Desktop/admin-control-panel/src/app/store/auth'

./src/app/store/auth/auth.state.ts:10:0-95 - Error: Module not found: Error: Can't resolve '@ngxs-labs/actions-executing/lib/actions-executed.state' in '/Users/user/Desktop/admin-control-panel/src/app/store/auth'
```

How to fix this?

--- a/src/lib/actions-executed.state.ts
+++ b/src/lib/actions-executed.state.ts
@@ -26,8 +26,7 @@ export class ActionsExecutedState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    var state = getState();
-                    let count = state ? state[actionType] || 0 : 0;
+                    let count = getState()?.[actionType] ?? 0;
 
                     if (actionContext.status !== ActionStatus.Dispatched) {
                         count++;

--- a/src/lib/actions-executed.state.ts
+++ b/src/lib/actions-executed.state.ts
@@ -26,7 +26,8 @@ export class ActionsExecutedState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    let count = getState()[actionType] || 0;
+                    var state = getState();
+                    let count = state ? state[actionType] || 0 : 0;
 
                     if (actionContext.status !== ActionStatus.Dispatched) {
                         count++;

--- a/src/lib/actions-executed.state.ts
+++ b/src/lib/actions-executed.state.ts
@@ -26,7 +26,7 @@ export class ActionsExecutedState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    let count = getState()?.[actionType] ?? 0;
+                    let count = getState()?.[actionType] || 0;
 
                     if (actionContext.status !== ActionStatus.Dispatched) {
                         count++;

--- a/src/lib/actions-executing.state.ts
+++ b/src/lib/actions-executing.state.ts
@@ -26,8 +26,7 @@ export class ActionsExecutingState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    var state = getState();
-                    let count = state ? state[actionType] || 0 : 0;
+                    let count = getState()?.[actionType] ?? 0;
 
                     if (actionContext.status === ActionStatus.Dispatched) {
                         count++;

--- a/src/lib/actions-executing.state.ts
+++ b/src/lib/actions-executing.state.ts
@@ -26,7 +26,7 @@ export class ActionsExecutingState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    let count = getState()?.[actionType] ?? 0;
+                    let count = getState()?.[actionType] || 0;
 
                     if (actionContext.status === ActionStatus.Dispatched) {
                         count++;

--- a/src/lib/actions-executing.state.ts
+++ b/src/lib/actions-executing.state.ts
@@ -26,7 +26,8 @@ export class ActionsExecutingState implements NgxsOnInit, OnDestroy {
                         return;
                     }
 
-                    let count = getState()[actionType] || 0;
+                    var state = getState();
+                    let count = state ? state[actionType] || 0 : 0;
 
                     if (actionContext.status === ActionStatus.Dispatched) {
                         count++;

